packages/sdk/src/types/runtime/actor.ts

```diff
@@ -694,4 +684,50 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
                 .catch(reason => log.error('app', reason));
         }
     }
+
+    private generateColliderGeometry(
+        colliderType: 'box' | 'sphere',
+        center?: Vector3Like,
+        size?: number | Vector3Like
+    ): ColliderGeometry {
+        switch (colliderType) {
+            case 'box':
+                return {
+                    colliderType: 'box',
+                    center,
+                    size: size as Partial<Vector3Like>
+                } as BoxColliderGeometry;
+            case 'sphere':
+                return {
+                    colliderType: 'sphere',
+                    center,
+                    radius: size as number
+                } as SphereColliderGeometry;
+
+            default:
+                log.error(null,
+                    'Trying to enable a collider on the actor with an invalid collider geometry type.' +
+                    `Type given is ${colliderType}`);
+
+                return undefined;
+        }
+    }
+
+    private _setCollider(collider: Partial<ColliderLike>) {
+        if (this._collider) {
+            unobserve(this._collider);
+            this._collider = undefined;
+        }
+
+        if (!this._collider) {

```

Comment:
>if (!this._collider) { [](start = 8, length = 22)

I think this check is unnecessary. this._collider will always be undefined here.

Reply:
Yeah.  That was a left over check from before changing the enable to set paradigm.  Removing it.